### PR TITLE
Feedback links

### PR DESCRIPTION
**Pull request by cmuld3r**
- Added a feedback button on the search page per #16 
- modified text message wording per #17

### CODE REVIEW

File: app/models/profile.rb [hunk 1]
```diff
@@ -13,13 +13,12 @@ def contact(other_profile)
 
     if !other_profile.driver?
       msg = %[
-        RideWithBernie: #{first_name} at #{phone} offered to drive you to #{event_title}, 
-        coming from #{location} with #{seats} seat#{'s' if seats != 1} available.
+        RideWithBernie: #{first_name} at #{phone} from #{location} offered to drive you to #{event_title}. Can you go?
```
Comment (kyleschmolze) [inline app/models/profile.rb:16]: I think we're gonna get a lot of people replying to this text message directly if we end with a question! I'm gonna make some quick changes.

Comment (kyleschmolze) [inline app/models/profile.rb:16]: Nvm! Didn't see this convo was already happening in #17. I'll keep quiet until that is resolved there.

**Comment by kyleschmolze** [comment]
Hey @cmuld3r what do you think about my small change from "Send us feedback" to "Send app feedback". I'm thinking it might not be obvious who "we" are! But I like the placement, and I also added a button to the `events/build` success page.

**Comment by cmuld3r** [comment]
hm, 'app' can have so many contexts, can we just remove the word in the
middle, or rephrase? like 'send feedback' or 'submit feedback' or something
else?

Christopher Mulder
chris@slonikstudios.com
818.297.2821 <+18182972821>
http://slonikstudios.com/

**Comment by kyleschmolze** [comment]
Yeah, I like "Submit feedback" best, as I've def seen that around so I think people will know what it means.

**Comment by clarissa1287** [comment]
Good thinking with the feedback. I approve. :+1:

**Pull request merged**